### ISSUE TITLE
Final Submission Folder Disappeared

### ISSUE THREAD

**Issue by bryankwe** [nus-cs2030/2122-s2#610]
Hi all, did the Final Submission Folder in CodeCrunch disappear for you guys too? 

![image](https://user-images.githubusercontent.com/97665622/165658907-79652fd2-502e-4a07-8978-86290be055b1.png)

Anyways atb everyone !

**Comment by AaronTay21** [comment]
Yes me too!

**Comment by coffeecode2** [comment]
same here, but I think they will probably open it up again in awhile.
atb!!

**Comment by calvinseptyanto** [comment]
me too hahaha, think will be up again soon!

**Comment by leontwl** [comment]
It will reopen at 1 pm if i remember correctly

**Comment by Nadiahhh** [comment]
Yup! If I'm not wrong, Prof stated in the email that the file will close today and reopen at 1pm :)

**Comment by Swagston20** [comment]
Prof say will reopen later
